mypy/report.py

```diff
@@ -144,6 +144,58 @@ def on_finish(self) -> None:
 register_reporter('linecount', LineCountReporter)
 
 
+class AnyExpressionsReporter(AbstractReporter):
+    def __init__(self, reports: Reports, output_dir: str) -> None:
+        super().__init__(reports, output_dir)
+        self.counts = {}  # type: Dict[str, Tuple[int, int]]
+        stats.ensure_dir_exists(output_dir)
+
+    def on_file(self,
+                tree: MypyFile,
+                type_map: Dict[Expression, Type],
+                options: Options) -> None:
+        visitor = stats.StatisticsVisitor(inferred=True, typemap=type_map, all_nodes=True)
+        tree.accept(visitor)
+        num_total = visitor.num_imprecise + visitor.num_precise + visitor.num_any
+        if num_total > 0:
+            self.counts[tree.fullname()] = (visitor.num_any, num_total)
+
+    def on_finish(self) -> None:
+        total_any = sum(num_any for num_any, _ in self.counts.values())
+        total_expr = sum(total for _, total in self.counts.values())
+        total_coverage = (float(total_expr - total_any) / float(total_expr)) * 100
+
+        any_column_name = "Anys"
+        total_column_name = "Exprs"
+        file_column_name = "Name"
+        # find the longest filename all files
+        name_width = max([len(file) for file in self.counts] + [len(file_column_name)])
+        # totals are the largest numbers in their column - no need to look at others
+        any_width = max(len(str(total_any)) + 3, len(any_column_name))
```

Comment:
This is the minimum distance between numbers in two columns. 

Hopefully, this is more clear after the new commit.